packages/client-sdk-nodejs/src/config/retry/fixed-timeout-retry-strategy.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  DeterminewhenToRetryRequestProps,
+  RetryStrategy,
+} from './retry-strategy';
+import {EligibilityStrategy} from './eligibility-strategy';
+import {
+  MomentoLoggerFactory,
+  MomentoLogger,
+  DefaultMomentoLoggerFactory,
+} from '../..';
+import {DefaultStorageEligibilityStrategy} from './storage-default-eligibility-strategy';
+
+export interface FixedTimeoutRetryStrategyProps {
+  loggerFactory: MomentoLoggerFactory;
+  eligibilityStrategy?: EligibilityStrategy;
+
+  // Retry request after a fixed time interval (defaults to 100ms)
+  retryDelayIntervalMillis?: number;
+
+  // Number of milliseconds the client is willing to wait for response data to be received before retrying (defaults to 1000ms). After the overarching GRPC config deadlineMillis has been reached, the client will terminate the RPC with a Cancelled error.
+  responseDataReceivedTimeoutMillis?: number;
+}
+
+export class FixedTimeoutRetryStrategy implements RetryStrategy {
```

Comment:
we need to add this to the exports in the main `index.ts` so that users can import it.